cloud_init.tf

```diff
@@ -6,14 +6,9 @@ data "template_file" "setup" {
     elasticsearch_log_dir             = "${var.elasticsearch_log_dir}"
     es_cluster_name                   = "${var.service_name}-${var.environment}-elasticsearch"
     es_allowed_urls                   = "${var.es_allowed_urls}"
-    aws_security_group                = "${aws_security_group.elasticsearch.id}"
```

Comment:
why is this being deleted? security groups are basically firewall rules, so it seems reasonable to accept this as a parameter